### ISSUE TITLE
Command 'Dark SynthWave '84: Enable Neon Dreams' resulted in an error (command 'darkSynthwave84.enableNeon' not found)

### ISSUE THREAD

**Issue by Nezteb** [brainomite/dark-synthwave-vscode#2]
<img width="275" alt="Screen Shot 2021-12-16 at 12 36 09 PM" src="https://user-images.githubusercontent.com/3588798/146429637-2cb942fd-f76d-4c68-b19d-cdbdfcc13649.png">

macOS 11.6.1
VS Code 1.63.2 (Universal)

I'm able to enable the neon dreams in [the normal SynthWave '84 theme](https://marketplace.visualstudio.com/items?itemName=RobbOwen.synthwave-vscode), but not this dark version.

**Comment by sfksuperman** [comment]
![image](https://user-images.githubusercontent.com/34414624/147368637-52c7e28c-b786-4e31-91e3-0e14c0485be6.png)

Any quick resolution?

**Comment by brainomite** [comment]
Thank you for bringing this to my attention, I'll look into this in a few days, if not sooner.

**Comment by chenzilu1990** [comment]
> ![image](https://user-images.githubusercontent.com/34414624/147368637-52c7e28c-b786-4e31-91e3-0e14c0485be6.png)
> 
> Any quick resolution?

the same issues

**Comment by MapleDemon** [comment]
Just need to modify the file in the plug-in directory "./src/extension.js" and use VSCode to shorten the content of the file to use TAB. The value of TAB is 4 Spaces, so it can be used normally. Besides, I don't understand why the author set the code to pop up an annoying blank page every time when I open VSCode...

**Comment by TheCelebrimbor** [comment]
![image](https://user-images.githubusercontent.com/13927773/149252256-62894d37-4a3b-47bc-bfa6-9320694a3887.png)

I think this is the root cause, the extension is not activated at all. @brainomite Please look into this.

**Comment by ElVictorious** [comment]
> Thank you for bringing this to my attention, I'll look into this in a few days, if not sooner.

I do like your extension colors? where you able to look into it?

**Comment by alizehctl** [comment]
Getting the same issue on Linux (Fedora 36). I ran `sudo chown -R $(whoami) /usr/share/code` if that info is useful at all

**Comment by brainomite** [comment]
> Just need to modify the file in the plug-in directory "./src/extension.js" and use VSCode to shorten the content of the file to use TAB. The value of TAB is 4 Spaces, so it can be used normally. Besides, I don't understand why the author set the code to pop up an annoying blank page every time when I open VSCode...

I removed the pop-up - but can you elaborate about shortening the content?

**Comment by brainomite** [comment]
> Just need to modify the file in the plug-in directory "./src/extension.js" and use VSCode to shorten the content of the file to use TAB. The value of TAB is 4 Spaces, so it can be used normally. Besides, I don't understand why the author set the code to pop up an annoying blank page every time when I open VSCode...

@MapleDemon - I resolved the pop-up
Can you elaberate further on the TAB? I am not following on the cause of the issue in your opinion.

**Comment by brainomite** [comment]
@TheCelebrimbor Did it ever work for you?

**Comment by brainomite** [comment]
> Getting the same issue on Linux (Fedora 36). I ran `sudo chown -R $(whoami) /usr/share/code` if that info is useful at all

@aadilayub did it ever work for you?

**Comment by TheCelebrimbor** [comment]
@brainomite Nah, I switched to a normal theme

**Comment by alizehctl** [comment]
@brainomite I cloned [this PR](https://github.com/brainomite/dark-synthwave-vscode/pull/4) to my `.vscode/extensions` folder and it worked for me. Since you've merged it now I think the error should be gone on the main branch as well.

**Comment by scarlion1** [comment]
When I try to use it now, first the error is:

> Cannot read properties of undefined (reading 'filename')

with more discussion on that with [Robb's extension](https://github.com/robb0wen/synthwave-vscode/issues/317).

the patches to Robb's `extension.js` don't exactly align with this version.  I tried replacing [this block](https://github.com/brainomite/dark-synthwave-vscode/blob/main/src/extension.js#L27-L41)  with the code mentioned [here](https://github.com/robb0wen/synthwave-vscode/issues/317#issuecomment-2351804426).  That led to more errors including the one mentioned in this issue (except without "Dark"), and now I've realized [Robb's current extension.js](https://github.com/robb0wen/synthwave-vscode/blob/master/src/extension.js) is much different than this one.

  Is glow still working for any of you?  I must have it 🤤

**Comment by scarlion1** [comment]
Ok I can't believe this worked but I thought I'd see if there was a way to install Robb's Synthwave then copy your palette into it, and all I had to do was copy your [themes/synthwave-color-theme.json](https://github.com/brainomite/dark-synthwave-vscode/blob/main/themes/synthwave-color-theme.json) into my `extensions/robbowen.synthwave-vscode-0.1.17/themes/` dir (overwriting his version) and restart codium.

_Voila!_
![Image](https://github.com/user-attachments/assets/5e243e77-f59d-4caa-b4df-2bf05d065f7b)
How lucky is that?

**Comment by KGlol** [comment]
> Ok I can't believe this worked but I thought I'd see if there was a way to install Robb's Synthwave then copy your palette into it, and all I had to do was copy your [themes/synthwave-color-theme.json](https://github.com/brainomite/dark-synthwave-vscode/blob/main/themes/synthwave-color-theme.json) into my `extensions/robbowen.synthwave-vscode-0.1.17/themes/` dir (overwriting his version) and restart codium.
> 
> _Voila!_ ![Image](https://github.com/user-attachments/assets/5e243e77-f59d-4caa-b4df-2bf05d065f7b) How lucky is that?

I'm having the same issue as you, but clearly you're mixing two themes here – this isn't a pure dark theme variant. Take a look at the author's screenshot for reference.

**Comment by scarlion1** [comment]
> I'm having the same issue as you, but clearly you're mixing two themes here – this isn't a pure dark theme variant. Take a look at the author's screenshot for reference.

Yes, I realized that later but, I was so enamored with my success, I barely noticed.  Gotta take what I can get these days y'know? 😛

You mean the purplish background #18131f and line number background #292137?  I tried to figure out where those were coming from but couldn't.  They seem to be a function of the Neon Dreams, though, because the background is black without it enabled.  They also increase in brightness with the Neon Dreams' brightness, which I had up high for the screenshot, 95% I believe.

What'd you end up doing?

Edit: Actually, the author's screenshots have some variation as well, registering the background as #060606 without the glow and slightly brighter at #131313 with the glow.

**Comment by TheCelebrimbor** [comment]
@scarlion1 @KGlol  I fixed the issues here - https://marketplace.visualstudio.com/items?itemName=Celebrimbor.new-dark-synthwave-vscode, You can try it out if it works. 

Make sure to uninstall other stuff first
